src/praisonai/praisonai/n8n/converter.py

```
@@ -0,0 +1,280 @@
+        return connections
+    
+    def _create_switch_node(self, route_config: Dict[str, Any]) -> N8nNode:
+        """Convert route configuration to n8n Switch node."""
+        self.node_counter += 1
+        
+        rules = []
+        for route_key in route_config.keys():
+            if route_key != "default":
+                rules.append({
+                    "operation": "equal",
+                    "value1": f"={{$json.condition}}",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)